sdk/monitor/Azure.Monitor.Query/api/Azure.Monitor.Query.netstandard2.0.cs

```diff
@@ -132,6 +132,40 @@ public partial class MetricsQueryOptions
 }
 namespace Azure.Monitor.Query.Models
 {
+    [System.Runtime.InteropServices.StructLayoutAttribute(System.Runtime.InteropServices.LayoutKind.Sequential)]
```

Comment:
Do you need to set the flag for this library as well? I don't think we want these types.